app/controllers/items_controller.rb

```diff
@@ -19,7 +21,18 @@ def create
   end
 
   def show
-    @item = Item.find(params[:id])
+  end
+
+  def edit
+  end
+
+  def update
+    @item.update(item_params)
+    if @item.save
```

Comment:
```suggestion
    if @item.update(item_params)
```
saveメソッドは不要ですので、上記のように変更しましょう。